app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/DatasetsOverviewActivityTest.kt

```diff
@@ -53,15 +64,44 @@ class DatasetsOverviewActivityTest {
 
     @Test
     fun fragmentIsDisplayedWhenActivityIsLaunched() {
-        assertDisplayed(R.id.dataOverview_fragment)
-        assertDisplayed(R.id.dataOverviewButton)
+        assertDisplayed(R.id.datasetsOverview_fragment)
+        assertDisplayed(R.id.datasetsOverviewButton)
     }
 
     @Test
-    fun rightActivityIsStartedAfterCreateButton() {
-        onView(withId(R.id.dataOverviewButton)).perform(click())
+    fun rightActivityIsStartedAfterCreateButtonIsClicked() {
+        onView(withId(R.id.datasetsOverviewButton)).perform(click())
         intended(hasComponent(CategoriesEditingActivity::class.java.name))
+        onView(withId(R.id.button_submit_list)).perform(click())
+    }
+
+    @Ignore("TODO: enable when DatasetListWidget and DisplayDatasetActivity use the same DB")
```

Comment:
This has been done in https://github.com/HumanLearning2021/HumanLearningApp/pull/103
```suggestion
```